### PR TITLE
Upgrade nns-js

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
We need nicely formatted bitcoin proposals

# Changes
Bump nns-js

# Tests

### CODE REVIEW

File: frontend/svelte/package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
   "dependencies": {
     "@dfinity/agent": "^0.11.1",
     "@dfinity/auth-client": "^0.11.1",
-    "@dfinity/nns": "nightly",
```
Comment (lmuntaner) [inline frontend/svelte/package.json:66]: we revert this change manually

**Review by lmuntaner — CHANGES_REQUESTED** [review]

File: frontend/svelte/package.json (hunk 1 above)
Comment (bitdivine) [reply frontend/svelte/package.json]: Good spot.  I have changed the command to `npm update ...` so that it does the right thing without manual changes.

**Review by lmuntaner — APPROVED** [review]
Thanks!
